meteor/client/ui/RundownList.tsx

```diff
@@ -297,6 +297,7 @@ export const RundownList = translateWithTracker(() => {
 											<span>{t('On Air Start Time')}</span>
 											<span>{t('Duration')}</span>
 											<span>{t('Last updated')}</span>
+											<span>{t('Shelf Layout')}</span>
```

Comment:
Given that at this stage, the component already knows if there are possibly any Rundown Layouts exposed to be used for the button, I'd like this column to be conditional, so that it doesn't show up if `rundownLayouts.length === 0`.